src/frontend/Frontend.ml

```diff
@@ -52,20 +53,24 @@ struct
 end
 and Runner :
 sig
-  val execute : per_process_opt : Contextual.per_process option -> mlconf : ML.mlconf ->
-    mlcmd : ML.mlcmd ML.info -> ResEnv.t * Contextual.per_process
+  val execute
+    : per_process_opt : Contextual.per_process option
+    -> mlconf : ML.mlconf
+    -> mlcmd : ML.mlcmd ML.info
+    -> ResEnv.t * Contextual.per_process
 end =
 struct
-  open ML
-  open Contextual
+  module M = Monad.Notation (Contextual)
+  open ML open M open Contextual
+
   let execute ~per_process_opt ~mlconf ~mlcmd =
     try
       run ~per_process_opt ~mlconf begin
-        bind (Elab.eval_cmd mlcmd.con) @@ fun _ ->
-        bind (report_unsolved ~loc:mlcmd.span) @@ fun _ ->
-        bind resolver @@ fun res ->
-        bind get_per_process @@ fun per_process ->
-        ret (res, per_process)
+        Elab.eval_cmd mlcmd.con >>
+        report_unsolved ~loc:mlcmd.span >>
+        resolver >>= fun res ->
+        get_per_process <<@> fun per_process ->
+          res, per_process
```

Comment:
Why do we want extra indentation in the last line?

Reply:
Personally, I don't want the extra indentation. But the indentation policy is to do whatever `ocp-indent` does.